homeassistant/components/frontend/__init__.py

```diff
@@ -447,8 +449,13 @@ def update_theme_and_fire_event() -> None:
                     themes[name].get(PRIMARY_COLOR, DEFAULT_THEME_COLOR),
                 ),
             )
+            MANIFEST_JSON.update_key(
+                "background_color",
+                themes[name].get(PRIMARY_BACKGROUND_COLOR, DEFAULT_BACKGROUND_COLOR),
+            )
         else:
             MANIFEST_JSON.update_key("theme_color", DEFAULT_THEME_COLOR)
+            MANIFEST_JSON.update_key("background_color", DEFAULT_BACKGROUND_COLOR)
```

Comment:
The default theme also has a dark mode, so we should not just set it to white (`#FFF`) when the theme is the default theme.